azure-pipelines.yml

```diff
@@ -118,7 +118,7 @@ jobs:
     parameters:
       jobTemplate: build-job.yml
       buildConfig: checked
-      ${{ if and(eq(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.DefinitionName'], 'coreclr-ci')) }}:
+      ${{ if and(in(variables['Build.Reason'], 'PullRequest'), in(variables['Build.DefinitionName'], 'coreclr-ci')) }}:
```

Comment:
I think of "in" as a superset of "eq". It makes it easier to extend it later. In general, it's hard for me to see why you would want to use "eq" instead of "in".